### PR TITLE
Build schedule which is required for time slicing demo

### PR DESCRIPTION
**Pull request by nishp1**
Update README so it can be built successfully locally.

### CODE REVIEW

**Comment by react-linter** [comment]
<details>
  <summary>Details of bundled changes.</summary>

  <p>Comparing: 7bcc0778fdd03fd0b8e9f2b4177829cd1ce31997...845ede2de6ca8ba73873ff4cdd0b88cdfdc993b5</p>


  
## schedule
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
schedule.development.js | n/a | n/a | 0 B | 19.17 KB | 0 B | 5.74 KB | UMD_DEV
**schedule.production.min.js** | **n/a** | **n/a** | **0 B** | **3.16 KB** | **0 B** | **1.53 KB** | **UMD_PROD**

  </details>
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: fixtures/unstable-async/time-slicing/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ There are also known bugs and inefficiencies in master so **don't use this fixtu
 # 1: Build react from source
 cd /path/to/react
 yarn
-yarn build dom,core,interaction,simple-cache-provider --type=NODE
```
Comment (plievone) [inline fixtures/unstable-async/time-slicing/README.md]: The same change could benefit the suspense demo readme too. And is there an interaction-package or was that an earlier name of schedule?

Comment (nishp1) [reply fixtures/unstable-async/time-slicing/README.md]: Updated suspense readme. Not 100% sure if interaction was renamed.

Comment (gaearon) [reply fixtures/unstable-async/time-slicing/README.md]: interaction is now part of schedule

**Comment by gaearon** [comment]
Thanks

**Pull request merged**